src/go/agwd/server/server.go

```diff
@@ -76,18 +75,15 @@ func cleanupUnixSocket(
 	}
 	logger.Warning().Printf("Removing existing socket file; previous unclean shutdown?")
 	if err := osRemove(path); err != nil {
-		return errors.Wrapf(err, "os.Stat(%s)", path)
+		return fmt.Errorf("os.Stat(%s): %w", path, err)
 	}
 
 	return nil
 }
 
 func cleanupUnixSocketOrDie(logger log.Logger, path string) {
 	if err := cleanupUnixSocket(logger, os.Stat, os.Remove, net.DialTimeout, path); err != nil {
-		panic(errors.Wrapf(
-			err,
-			"cleanupUnixSocket(logger=_, target.Endpoint=%s)",
-			path))
+		panic(fmt.Errorf("cleanupUnixSocket(logger=_, target.Endpoint=%s): %w", path, err))
```

Comment:
While in general cleanup is nice, we might not want to change this in the same commit and PR, as the PR is already big enough. (Also lines below.)

Reply:
This line and other instances of this in the file also change `errors.Wrapf` to `fmt.Errorf`. The reformatting was incidental.